src/app/page.tsx

```diff
@@ -333,7 +339,7 @@ export default function Home() {
     })
       .then((result) => console.log("Success. Result: ", result))
       .then(() => setProgressButton(true))
-      .catch(console.error);
+      .catch(console.error)
```

Comment:
This is where we can catch the error from the pipeline. For now, to fix the "images being generated" message showing up and not disappearing, we could change this to something like
`.catch(() => {console.error; setProgressButton(true)})`
which at least for me shows the process completed message on an error.

For future error handling, I think something like
`.catch((error) => handlePipelineError(error)})`
might work!